amazon-sns/4.5/modules/ROOT/pages/amazon-sns-connector-reference.adoc

```diff
@@ -42,6 +42,7 @@ Release Notes: xref:release-notes::connector/amazon-sns-connector-release-notes-
 | Secret Key a| String |  The secret key provided by Amazon. |  | x
 | Try Default AWSCredentials Provider Chain a| Boolean |  Set this field to `true` to obtain credentials from the AWS environment. See https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html[Supplying and Receiving AWS Credentials]. |  false |
 | Region Endpoint a| String |  Set the topic region endpoint. |  US_EAST_1 |
+| Custom Service Endpoint a| String |  Sets a custom service endpoint. Useful when a non-standard service endpoint is required, such as a VPC endpoint. |  |
```

Comment:
This is very picky, but since the bullet before this one starts with "Set," you should make them consistent. (I see "Sets" used later on.)